stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -69,7 +145,179 @@ private static String getIdAttribute(View element) {
 
   @Override
   public View getViewForHighlighting(Object element) {
-    return (View)element;
+    return (View) element;
+  }
+
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+
+    for (ViewCSSProperty property : getViewProperties()) {
```

Comment:
because this is a `List`, we can use the iterator-less pattern and save an allocation:
```
List<ViewCSSProperty> viewProperties = getViewProperties();
for (int i = 0, size = viewProperties.size(); i < size; ++i) {
  final ViewCSSProperty property = viewProperties.get(i);
  ...
}
```